src/main/scala/com/comcast/xfinity/sirius/api/SiriusConfiguration.scala

```diff
@@ -188,6 +188,17 @@ object SiriusConfiguration {
    */
   final val LOG_WRITE_CACHE_SIZE = "sirius.log.write-cache-size"
 
+  /**
+    * Whether or not to use a buffered reader when reading through the log
+    */
+  final val LOG_USE_READ_BUFFER = "sirius.log.read-buffer-enabled"
+
+  /**
+    * Indicates the size of the buffer in bytes when reading events from the log.
+    * The default buffer size is 8192 bytes. (int)
+    */
+  final val LOG_READ_BUFFER_SIZE = "sirius.log.read-buffer-size"
```

Comment:
Addendum nitpick: we should indicate the units in the prop name/values. like `final val LOG_READ_BUFFER_SIZE_BYTES = "sirius.log.read-buffer-size-bytes"`